.buildkite/pull-requests.json

```diff
@@ -25,12 +25,12 @@
             "set_commit_status": true,
             "build_on_commit": true,
             "build_on_comment": true,
-            "trigger_comment_regex": "^(?:(?:buildkite\\W+)?(?:build|test)\\W+(?:this|it))|^/test filebeat$",
-            "always_trigger_comment_regex": "^(?:(?:buildkite\\W+)?(?:build|test)\\W+(?:this|it))|^/test filebeat$",
+            "trigger_comment_regex": "^/test filebeat$|^/test filebeat for arm$|^/test filebeat for macos$|^/filebeat extended tests$|^/filebeat for windows$",
+            "always_trigger_comment_regex": "^/test filebeat$|^/test filebeat for arm$|^/test filebeat for macos$|^/filebeat extended tests$|^/filebeat for windows$",
```

Comment:
**nit**: regex can be simplified with some groups and `?`:

```suggestion
            "trigger_comment_regex": "^/test filebeat(for (arm|macos|windows))?$|^/filebeat extended tests$",
            "always_trigger_comment_regex": "^/test filebeat(for (arm|macos|windows))?$|^/filebeat extended tests$",
```

Maybe `^/filebeat extended tests$` could be changed to `/test filebeat for extended support`? This could help with

```
"trigger_comment_regex": "^/test filebeat(for (arm|macos|windows|extended support))?$",
"always_trigger_comment_regex": "^/test filebeat(for (arm|macos|windows|extended support))?$",
```

Reply:
Thanks, changed!